cmd/issuer-rest/static/receiveVC.html

```diff
@@ -216,12 +220,16 @@ <h1 class="w-full my-2 text-3xl font-bold leading-tight text-center text-gray-80
             $("#success-alert").hide()
         } else {
             $("#error-alert").hide()
+            $("#generated-cred").hide()
+            $("#main-section").addClass("py-48").removeClass("py-8")
```

Comment:
what is this doing ? add and removeClass

Reply:
I put this in because if I just use py-48 always on the #main-section element, then the generated credential appears shifted downwards on the page. And if I just use py-8 always, then when the success message is showing, the navbar is really tall. 
![Screenshot from 2020-09-30 14-02-02](https://user-images.githubusercontent.com/42557011/94843579-e2f6fc80-03ea-11eb-9e05-1443db42ac76.png)
![Screenshot from 2020-09-30 15-12-32](https://user-images.githubusercontent.com/42557011/94843598-e8ecdd80-03ea-11eb-8fe4-e81f51eb109d.png)

Please let me know if there is a better solution for this.